packages/client/js-client.web.standalone/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import { FluencePeer } from '@fluencelabs/js-peer/dist/js-peer/FluencePeer';
+import { MarineBasedAvmRunner } from '@fluencelabs/js-peer/dist/js-peer/avm';
+import { MarineBackgroundRunner } from '@fluencelabs/js-peer/dist/marine/worker';
+import { checkConnection, marineLogFunction } from '@fluencelabs/js-peer/dist/js-peer/utils';
+import { InlinedWorkerLoader, InlinedWasmLoader } from '@fluencelabs/js-peer/dist/marine/deps-loader/common';
+
+export const makeDefaultPeer = () => {
+    const workerLoader = new InlinedWorkerLoader('___worker___');
+    const controlModuleLoader = new InlinedWasmLoader('___marine___');
+    const avmModuleLoader = new InlinedWasmLoader('___avm___');
+
+    const marine = new MarineBackgroundRunner(workerLoader, controlModuleLoader, marineLogFunction);
+    const avm = new MarineBasedAvmRunner(marine, avmModuleLoader, undefined);
+    return new FluencePeer(marine, avm);
+};
+
+// @ts-ignore
+globalThis.defaultPeer = makeDefaultPeer();
+
+// TODO! remove
```

Comment:
please do remove, or if this hack is still necessary - please add a task and link it here

Reply:
I've updated the todo with more detailed info